src/main/app/claims/responseModelConverter.ts

```diff
@@ -169,12 +179,65 @@ export class ResponseModelConverter {
       } as DefendantEvidence,
       defendant: this.convertPartyDetails(draft.defendantDetails),
       paymentIntention: draft.partialAdmission.paymentIntention && this.convertPaymentIntention(draft.partialAdmission.paymentIntention),
-      freeMediation: FreeMediationUtil.convertFreeMediation(draft.freeMediation),
+      freeMediation: this.convertFreeMediation(mediationDraft, draft),
+      mediationPhoneNumber: this.convertMediationPhoneNumber(mediationDraft, draft, claim),
+      mediationContactPerson: this.convertMediationContactPerson(mediationDraft, draft, claim),
       statementOfMeans: this.convertStatementOfMeans(draft),
       statementOfTruth: this.convertStatementOfTruth(draft)
     }
   }
 
+  private static convertFreeMediation (mediationDraft: MediationDraft, draft: ResponseDraft): YesNoOption {
+    if (FeatureToggles.isEnabled('mediation')) {
+      return FreeMediationUtil.convertFreeMediation(mediationDraft.youCanOnlyUseMediation)
+    } else {
+      return FreeMediationUtil.convertFreeMediation(draft.freeMediation)
+    }
+  }
+
+  private static convertMediationPhoneNumber (mediationDraft: MediationDraft, draft: ResponseDraft, claim: Claim): string {
+    if (FeatureToggles.isEnabled('mediation')) {
+
+      if (mediationDraft.canWeUseCompany) {
+        if (mediationDraft.canWeUseCompany.option === FreeMediationOption.YES) {
+          return mediationDraft.canWeUseCompany.mediationPhoneNumberConfirmation
+        } else {
+          return mediationDraft.canWeUseCompany.mediationPhoneNumber
+        }
+      }
+
+      if (mediationDraft.canWeUse) {
```

Comment:
this should be an else if as it shouldn't be possible to have both canWeUse and canWeUseCopmany

Reply:
done